apps/src/applab/setPropertyDropdown.js

```diff
@@ -42,6 +42,7 @@ var PROP_INFO = {
   // it won't show up in the dropdown.
   pictureImage: { friendlyName: 'image', internalName: 'picture', type: 'string' },
   picture: { friendlyName: 'picture', internalName: 'picture', type: 'string', alias: true },
+  iconColor: { friendlyName: 'icon-color', internalName: 'icon-color', type: 'string' },
```

Comment:
How are `friendlyName` and `internalName` used?  It's unclear to me whether they should be different; on the one hand you have `group-id/groupId` and `text-color/textColor` but you see the opposite in `width/style-width` and `image/screen-image`.